kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm-ui/src/main/java/org/kie/server/remote/rest/jbpm/ui/ImageResource.java

```diff
@@ -74,11 +74,13 @@ public ImageResource(ImageServiceBase imageServiceBase, KieServerRegistry contex
     @Produces({MediaType.APPLICATION_SVG_XML})
     public Response getProcessImage(@javax.ws.rs.core.Context HttpHeaders headers,
             @ApiParam(value = "container id that process definition belongs to", required = true, example = "evaluation_1.0.0-SNAPSHOT") @PathParam(CONTAINER_ID) String containerId,
-            @ApiParam(value = "identifier of the process definition that image should be loaded for", required = true, example = "evaluation") @PathParam(PROCESS_ID) String processId) {
+            @ApiParam(value = "identifier of the process definition that image should be loaded for", required = true, example = "evaluation") @PathParam(PROCESS_ID) String processId,
+            @ApiParam(value = "svg width", required = false, example = "100%") @QueryParam(SVG_WIDTH) @DefaultValue("") String svgWidth,
+            @ApiParam(value = "svg height", required = false, example = "500") @QueryParam(SVG_HEIGHT) @DefaultValue("") String svgHeight) {
```

Comment:
Why we have one value in % and the other not?

Reply:
yes, I changed that to a numerical value in both parameters. My intention was show that this parameter could be in any format that is allowed to width and height. But it's true that looked a bit strange. Fixed now. Thanks!